app/browser-sync.ts

```diff
@@ -83,6 +83,29 @@ export class BrowserSync {
 
     return result;
   }
+
+  forkNewDriverInstance(useSameUrl?: boolean, copyMockModules?: boolean, copyConfigUpdates?: boolean) {
+    const newBrowser = this.getBrowser().forkNewDriverInstance(useSameUrl, copyMockModules, copyConfigUpdates);
+
+    const waitForAngularEnabled = this.waitForAngularEnabled();
+
+    //Use the runner class to reset global variables after spawning the new instance
+    const runner = new Runner(this.getProcessedConfig());
+    runner.setupGlobals_(newBrowser);
```

Comment:
I agree this is a reasonable default. And it's exactly our use-case.

I do think the consistency with Protractor is an issue though. I think it's especially confusing to have a method with the same name but different behavior, so we'll go ahead and rename this method.